manager/manager/typeresolver.go

```diff
@@ -48,10 +53,12 @@ func NewTypeResolver() TypeResolver {
 	client.Timeout = timeout
 	ret.getter = util.NewHTTPClient(3, client, util.NewSleeper())
 	ret.maxUrls = maxURLImports
+	ret.re = regexp.MustCompile("github.com/(.*)/(.*)/(.*)/(.*):(.*)")
```

Comment:
Yeah, this is what the registry spec says. I would like to leave out the semver from this change since it's big as it is.